src/Microsoft.Health.Dicom.Web/Startup.cs

```diff
@@ -57,9 +67,42 @@ public virtual void ConfigureServices(IServiceCollection services)
     // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
     public virtual void Configure(IApplicationBuilder app)
     {
-        app.UseDicomServer();
+        EnsureArg.IsNotNull(app, nameof(app));
+
+        app.UseQueryStringValidator();
+
+        app.UseHttpsRedirection();
+
+        app.UseCachedHealthChecks(new PathString(KnownRoutes.HealthCheck));
+
+        // Update Fellow Oak DICOM services to use ASP.NET Core's service container
+        DicomSetupBuilder.UseServiceProvider(app.ApplicationServices);
+
+        IOptions<FeatureConfiguration> featureConfiguration = app.ApplicationServices.GetRequiredService<IOptions<FeatureConfiguration>>();
+        if (featureConfiguration.Value.EnableOhifViewer)
+        {
```

Comment:
I think the contents of this if-block could be in an extension method so callers could just do:

```csharp
if (featureConfiguration.Values.EnableOhifViewer)
{
    app.UseOhifViewer()
}
```